bindings/python/docs/environment.yml

```diff
@@ -0,0 +1,8 @@
+name: mongo_arrow_documentation
+channels:
+  - conda-forge
+dependencies:
+- python=3.9
+- sphinx
+- libbson
```

Comment:
no longer needed?

Reply:
We are still using it, as the "system" `libbson`

Reply:
Thanks!